lib/discourse_plugin_registry.rb

```diff
@@ -68,6 +68,7 @@ def self.define_filtered_register(register_name)
   define_register :vendored_pretty_text, Set
   define_register :vendored_core_pretty_text, Set
   define_register :seedfu_filter, Set
+  define_register :permitted_bulk_action_parameters, Set
```

Comment:
Can we move this to `define_filtered_register`. That way, it will automatically remove the item from the register when the plugin is disabled. There is a little documentation at the top of this file about how it all works.